src/platforms/php/guides/symfony/config.mdx

```diff
@@ -54,6 +49,12 @@ sentry:
     request: 1
     subrequest: 1
     console: 1
+services:
```

Comment:
I would add a small comment here to say that this piece is suggested but not required. It makes duplicate reporting of events if `register_error_listener` is still set to `true`.

Reply:
Agreed, but it should be underneath where all the other options are being explained.

Also mark this as being in `config/services.yml` file. There are all levels of developers out there and spelling it out is sometimes required and makes for a much less frustrating experience for everyone.

Reply:
would not say this need to live in the `config/services.yaml` its okay to be in the `config/packages/sentry.yaml` because it is part of the sentry configuration. Its not common that bundles the define the service definitions for there package in there config file. So you actually see that this service definition is only here because of sentry and not part of your application service definitions.

https://github.com/symfony/recipes-contrib/search?q=services

Reply:
Okay, thanks for pointing it out. I moved this to a separate paragraph, let me know if it is accurate now.